### PR TITLE
Improve ObservationView handling of SBP 2.x

### PR DESCRIPTION
**Pull request by ljbade**
This tweaks the ObservationView handling of SBP 2.x to make the flags easier to understand.
- adds labels after the flags value to indicate it's meaning, e.g. `PR CP` means pseudorange and carrier phase valid
- sets invalid fields to zero, to make it obvious the value is invalid and should not be interpreted

It also removes the dead RINEX recording code.

@denniszollo Can you review?

### CODE REVIEW

File: piksi_tools/console/observation_view.py [hunk 1]
```diff
@@ -198,80 +126,88 @@ def obs_packed_callback(self, sbp_msg, **metadata):
...
+      if (flags & 0x08):
+        flags_string += "MD "
+
+      pr = float(o.P) / divisor
+      cp = float(o.L.i) + float(o.L.f) / (1 << 8)
+      cn0 = float(o.cn0) / 4
+
+      # Sets invalid values to zero
+      # TODO(Leith): it would be better if the cell was blank when invalid
+      if sbp_msg.msg_type not in [SBP_MSG_OBS_DEP_A, SBP_MSG_OBS_DEP_B, SBP_MSG_OBS_DEP_C]:
```
Comment (denniszollo) [inline piksi_tools/console/observation_view.py]: Lets send out the raw value even if invalid.  Doesn't help a customer to think that invalid values will always be 0

Comment (denniszollo) [inline piksi_tools/console/observation_view.py]: If we want to hide invalid values, I think we have to change every column to a string and use the "EMPTY_STR" global when no data exists.  That is currently set to be "--".

Comment (ljbade) [inline piksi_tools/console/observation_view.py]: ok

**Comment by ljbade** [comment]
@denniszollo addressed the review comments, can you review again?

File: piksi_tools/console/observation_view.py [hunk 2]
```diff
@@ -36,37 +36,22 @@ class SimpleAdapter(TabularAdapter):
                ('Flags', 7)]
     font='courier'
     alignment='center'
```
Comment (denniszollo) [inline piksi_tools/console/observation_view.py]: we need these format strings in order to prevent round numbers from jumping alignment

**Comment by denniszollo** [comment]
LGTM - merge provided travis passes linux.

**Pull request merged**